content/docs/user-guide/assets/pipeline/scan-directories.md

```diff
@@ -20,7 +22,40 @@ Asset processor monitors the following scan directories:
 1. **Engine & Editor** - The root directory and subdirectories of the engine and editor contain assets that are generally labeled critical and are required to run **O3DE Editor** and **Launcher**.
 
 1. **Intermediate Assets** - The root directory for intermediate assets which are output by other sources.  These files should be treated like the **Asset Cache**, they are generated files which should not be modified, and do not need to be tracked by source control.
-
-{{< note >}}
-Files in scan directories that Asset Processor doesn't recognize, such as license files, are ignored by Asset Processor. Ignored files aren't copied to the **Asset Cache**, and won't appear in **Asset Browser** in O3DE Editor.
-{{< /note >}}
+
+Additional scan directories can be added with registery keys. In most cases it's better to use gems as a way to store source assets in additional locations instead of adding scan folders by registry key.
```

Comment:
Per style guide, we are treating the word "Gem" as a proper noun.

```suggestion
Additional scan directories can be added with registry keys. In most cases it's better to use Gems as a way to store source assets in additional locations instead of adding scan folders by registry key.
```